PocketKit/Sources/Sync/Collection/CollectionUrlFormatter.swift

```diff
@@ -5,25 +5,38 @@
 import Foundation
 
 /// A formatter for collection URLs
+/// valid formats are
```

Comment:
Do you need to do this to Syndicated Items too? And Read url items?

Reply:
we only need this on collections because in certain situations we need to extract the slug from the path. The other two do not have a formatter like this one.